src/tv2-common/content/dve.ts

```diff
@@ -405,7 +405,7 @@ export function MakeContentDVE2<
 					const sourceInfoFull: SourceInfo = {
 						type: SourceLayerType.GRAPHICS,
 						id: 'full',
-						port: config.studio.AtemSource.DSK1F
+						port: config.dsk[1].Fill
```

Comment:
I don't think we should assume/force the first DSK to be the full graphics source. This would be the perfect use case for radio buttons in tables to select a single entry as a default, but for now maybe a checkbox and the first item seen with the "Full graphic source" checkbox checked will be used?

Reply:
I implemented this feature in https://github.com/olzzon/tv2-sofie-blueprints-inews/pull/18/commits/cdb7b06881665ce58e09990be10dc77a7b194d0f